utils/src/lib.rs

```diff
@@ -97,6 +97,26 @@ pub fn modulo(bytes: &[u8], n: u64) -> u64 {
     result
 }
 
+pub trait Serializable {
```

Comment:
If we don't implement this as part of the `Octets` trait, we can implement it for other types (like `u64`)